prac_06/guitars.py

```diff
@@ -0,0 +1,29 @@
+from prac_06.guitar import Guitar
+
+
+def main():
+    guitars = []
+    print("My Guitars")
+    name = input("Name: ")
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: "))
+        guitar_to_add = Guitar(name, year, cost)
+        guitars.append(guitar_to_add)
+        print(f"{guitar_to_add} added.")
+        # guitars.append(Guitar(name, year, cost))
+        name = input("Name: ")
+
+    guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
+    guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+    print("These are my Guitars:")
+
+    for i, guitar in enumerate(guitars, 1):
+        vintage_string = ""
+        if guitar.is_vintage():
+            vintage_string = " (vintage)"
+        print(f"Guitar {i}: {guitar.name:>20} ({guitar.year}), worth ${guitar.cost:10,.2f}{vintage_string}")
```

Comment:
You could use a method covered in class, such as a list comprehension, to determine the maximum length of a guitars name and use that for the formatting :)